src/showroom_connector/signals.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db.models.signals import post_save
+
+from user_preferences.models import UserPreferencesData, UserSettingsValue
+
+from .rq import enqueue
+from .sync import push_user
+
+
+def handle_post_save(sender, instance, *args, **kwargs):
+    if settings.SYNC_TO_SHOWROOM:
+        job_id = f'push_user_{instance.user.username}'
+        enqueue(job_id, push_user, user=instance.user)
+
+
+post_save.connect(
+    handle_post_save,
+    sender=UserPreferencesData,
+    dispatch_uid='post_save_userpreferencesdata',
+)
+post_save.connect(
+    handle_post_save,
+    sender=UserSettingsValue,
+    dispatch_uid='post_save_usersettingsvalue',
+)
+
+
+def create_user_preferences(sender, instance, *args, **kwargs):
```

Comment:
this signal shouldn't be part of `showroom_connector`, it should be in `user_preferences`